### PR TITLE
chore(build): add ConfigCheck workflow

### PR DESCRIPTION
**Pull request by KwadFan**
Add new check for PR or pushes,
this check inspects config files for the
bare minimum needed configuration.

This PR contains also preperation for
the new method to generate the JSON file
needed for the rpi-imager.

### CODE REVIEW

File: config/armbian/default [hunk 1]
```diff
@@ -37,3 +37,8 @@ export BASE_IMAGE_ENLARGEROOT
 export BASE_IMAGE_RESIZEROOT
 export BASE_RELEASE_COMPRESS
 export MODULES
+
+### JSON snipplet Setup
+### NOTE: Please see all config files for setup variables!!!
+# shellcheck disable=SC2034
+JSON_DESCRIPTION="A port of Armbian with preinstalled Mainsail for 3D Printer's"
```
Comment (meteyou) [inline config/armbian/default:44]:
```suggestion
JSON_DESCRIPTION="A port of Armbian with preinstalled Mainsail for 3D Printer's"
```
JSON_DESCRIPTION="A port of Armbian with preinstalled Mainsail for 3D printers"

File: config/orangepi/default [hunk 2]
```diff
@@ -37,3 +37,8 @@ export BASE_IMAGE_ENLARGEROOT
 export BASE_IMAGE_RESIZEROOT
 export BASE_RELEASE_COMPRESS
 export MODULES
+
+### JSON snipplet Setup
+### NOTE: Please see all config files for setup variables!!!
+# shellcheck disable=SC2034
+JSON_DESCRIPTION="A port of Orange Pi OS with preinstalled Mainsail for 3D Printer's"
```
Comment (meteyou) [inline config/orangepi/default:44]:
```suggestion
JSON_DESCRIPTION="A port of Orange Pi OS with preinstalled Mainsail for 3D Printer's"
```
JSON_DESCRIPTION="A port of Orange Pi OS with preinstalled Mainsail for 3D printers"

File: config/raspberry/default [hunk 3]
```diff
@@ -16,3 +16,8 @@ export BASE_IMAGE_ENLARGEROOT
 export BASE_IMAGE_RESIZEROOT
 export BASE_RELEASE_COMPRESS
 export MODULES
+
+### JSON snipplet Setup
+### NOTE: Please see all config files for setup variables!!!
+# shellcheck disable=SC2034
+JSON_DESCRIPTION="A port of Raspberry Pi OS with preinstalled Mainsail for 3D Printer's"
```
Comment (meteyou) [inline config/raspberry/default:23]:
```suggestion
JSON_DESCRIPTION="A port of Raspberry Pi OS with preinstalled Mainsail for 3D printers"
```

**Review by meteyou — CHANGES_REQUESTED** [review]

**Comment by KwadFan** [comment]
Will not be merged, decided against it.

**Pull request closed without merging**